prow/plank/controller.go

```diff
@@ -167,41 +160,42 @@ func (c *Controller) Sync() error {
 	close(pjCh)
 
 	errCh := make(chan error, len(pjs))
+	reportCh := make(chan kube.ProwJob, len(pjs))
```

Comment:
Yeah, on second thought I guess what you have is fine. We have at least one array of all the prowjobs floating around anyway, what harm could it be to add another :)